lib/net/ldap.rb

```diff
@@ -1323,4 +1317,17 @@ def new_connection
     }
     raise e
   end
+
+  # Normalize encryption parameter the constructor accepts, expands a few
+  # convenience symbols into recognizable hashes
+  def normalize_encryption(args)
+    return if args.nil?
+    return args if args.is_a? Hash
+
+    case method = args.to_sym
+    when :simple_tls, :start_tls
+      { :method => method, :tls_options => {} }
+    end
+  end
```

Comment:
Sorry, what exactly would you like me to do? The method should take 1 of 4 types of arguments (nil, string, symbol, or hash) and return either nil or a hash.